_docs/style/colors.md

```diff
@@ -11,20 +11,57 @@ status: complete
 {% include toc.html %}
 
 <section class="static-section"  markdown="1">
-### Primary Colors
 
-Primary colors should be used sparingly for primary and secondary actions,
-alerts, or data visualizations.
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-12-md hxSpan-12-lg" %}
+
+## The Helix color palette
+
+The image below contains all colors in the Helix color palette.
+
+{% figure [caption:"Helix Color Palette"] [class:"image bg-light border"] %}
+<embed src="{{site.url}}/assets/images/style/color-palette/color-palette.png" width="1900"/>
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+## Health status
+
+The health status family communicates the well-being of an infrastructure object,
```

Comment:
maybe but the examples in parentheses? The comma comma comma comma comma comma structure is confusing.

Reply:
gotcha. Reworked this, we'll see how ya like it!